packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -15,6 +15,7 @@ import type {
 } from './types/Simulation'
 import type { Session } from './types/Session'
 import type { Channel } from '../../utils/channel'
+import {getCookie} from '../../utils/getCookies'
```

Comment:
looks like the eslint/prettier is not working :(
```suggestion
import { getCookie } from '../../utils/getCookies'
```